config/config.go

```diff
@@ -99,6 +99,7 @@ type Config struct {
 	SMS                      SMSGlobalConfig  `json:"SMSGlobal"`
 	Webserver                WebserverConfig  `json:"Webserver"`
 	Exchanges                []ExchangeConfig `json:"Exchanges"`
+	Providers                []ProviderConfig `json:"Providers"`
```

Comment:
Need to come up with a more specific name, perhaps ForexProviders or ForexConversionProviders.

Reply:
ForexRateProviders?